CREATE_WORKLOAD_GUIDE.md

```diff
@@ -15,32 +15,87 @@ $ opensearch-benchmark create-workload \
 --target-hosts="<CLUSTER ENDPOINT>" \
 --client-options="basic_auth_user:'<USERNAME>',basic_auth_password:'<PASSWORD>'" \
 --indices="<INDICES TO GENERATE WORKLOAD FROM>" \
+--number-of-docs="<INDEX NAME>:<NUMBER OF DOCUMENTS>" 
 --output-path="<LOCAL DIRECTORY PATH TO STORE WORKLOAD>"
+--custom-queries="<FILE PATH OF CUSTOM QUERIES TO ADD FOR THE WORKLOAD EXECUTION>"
+--concurrent
+--threads="<NUMBER OF THREADS TO RUN THE INDEX EXTRACTION>"
+--bsize="<BATCH SIZE OF EVERY INDEX EXTRACTION QUERY WITHIN THREADS>"
+--custom_dump_query="<FILE PATH OF THE CUSTOM DOCUMENT SEARCH QUERY FOR INDEX EXTRACTION>"
 ```
 Note that:
-* `--indices` can be 1+ indices specified in a comma-separated list.
+* `--indices` can be 1+ indices specified in a comma-separated list in the format `index1,index2`
 * If the cluster uses basic authentication and has TLS enabled, users will need to provide them through `--client-options`.
+* `--number-of-docs`, `--concurrent`, `--threads`, `--bsize`, `--custom_dump_query` are OPTIONAL
+* `--number-of-docs` can be 1+ `index:num_docs` pairs in the format `index1:num_docs, index2:num_docs`
```

Comment:
`index1:num_docs, index2:num_docs` --> `index1:num_docs index2:num_docs` (no comma). See [this reference](https://github.com/opensearch-project/opensearch-benchmark/issues/289#issuecomment-1540789405)